docker/buildDocker.sh

```diff
@@ -1,13 +1,30 @@
 #!/bin/bash
 set -u
 
-jdkVersion=''
-bootDir=''
-buildVariant="hotspot"
+BOOTDIR=""
+VARIANT="hotspot"
 useEclipseOpenJ9DockerFiles=false
-cleanWorkspace=false
+CLEAN=false
+WORKSPACE=${PWD}
+JDK_VERSION=
+JDK_MAX=
+JDK_GA=
 
-#takes in all arguments to determine script options
+# shellcheck disable=SC2002 # Disable UUOC error
+setJDKVars() {
+	wget -q https://api.adoptopenjdk.net/v3/info/available_releases
+	JDK_MAX=$(cat available_releases \
```

Comment:
You could simplify this with awk which would reduce the number of processes by 3:
JDK_MAX=$(awk -F: '/tip_version/{gsub("[, ]","",$2); print$2}' < available_releases)